libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/NavigationCamera.java

```diff
@@ -67,6 +67,18 @@ public NavigationCamera(@NonNull MapboxMap mapboxMap, @NonNull MapboxNavigation
     initialize();
   }
 
+  /**
+   * Creates an instance of {@link NavigationCamera}.
+   *
+   * @param mapboxMap for moving the camera
+   * @since 0.15.0
+   */
+  //TODO Could it be package-private?
```

Comment:
I'm fine with either decision here.  If we decide to keep it `public`, let's make sure to also explain in the javadoc that the camera won't do anything until `addProgressChangeListener` is called. 

If we decide against the `public` API, I think we should remove the `camera` package altogether?

Reply:
Let's leave it `public` for now. We could revisit in a future PR.